exercises/chapter8/test/Main.purs

```diff
@@ -1,57 +1,27 @@
 module Test.Main where
 
 import Prelude
-import Data.AddressBook
-  ( examplePerson
-  , PhoneType(..)
-  )
-import Data.Array
-  ( filter
-  , null
-  )
-import Data.List
-  ( List(..)
-  , foldM
-  , (:)
-  )
+import Test.MySolutions
```

Comment:
There are warnings with unspecified imports, but those go away after the first build, and this reduces steps for a user when adding solutions functions.